website/source/docs/configuration/environment-variables.html.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: "docs"
+page_title: "Environment Variables"
+sidebar_current: "docs-config-environment-variables"
+description: |-
+  Something on Environment Variables!
```

Comment:
I'd just say something like "Details on the environment variables that can be used to configure various aspects of Terraform"